src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -195,7 +172,13 @@ export default class HaAutomationActionRow extends LitElement {
                 ></ha-icon-button>
               `
             : ""}
-          <ha-button-menu corner="BOTTOM_START" @action=${this._handleAction}>
+          <ha-button-menu
+            slot="header"
+            fixed
+            corner="BOTTOM_START"
+            @action=${this._handleAction}
+            @click=${preventDefault}
```

Comment:
Why is this needed?  Because of the nested buttons?

Reply:
Yes.